packages/launchpad/cypress/e2e/integration/project-setup.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { WIZARD_STEPS } from '@packages/types/src/constants'
+import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
+
+describe('Launchpad: Setup Project', () => {
+  it('no initial setup displays welcome page', () => {
+    cy.setupE2E('todos')
+    cy.visitLaunchpad()
+
+    cy.get('h1').should('contain', WIZARD_STEPS.find((s) => s.type === 'welcome').title)
+    cy.get('h2').should('contain', 'E2E Testing')
+    cy.get('h2').should('contain', 'Component Testing')
+  })
+
+  it('welcome page has button to learn about testing types', () => {
+    cy.setupE2E('todos')
+    cy.visitLaunchpad()
+
+    cy.get('button.hocus-link-default')
```

Comment:
I'd prefer to not depend on a specific css class when finding the button. We could just remove it, or possibly shorten it all to:

```
cy.contains('button', defaultMessages.welcomePage.review).click()
```

Reply:
Today I learned about `cy.contains()`! Will change to use that instead.